packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -412,7 +404,12 @@ def _load_font_into_web(loaded_face):
         if font_face_arguments not in _font_set:
```

Comment:
I think we should consider moving toward requiring people to explicitly install fonts and throwing an error otherwise, at least by default. If they can't figure out ahead of time what fonts to load, we could enable this approach as a compatibility mode fallback. I don't like this "guess what people want and do it on the fly", particularly as a default.

Reply:
Yes, I also feel like the current font loading system is somewhat problematic. It requires a fixed fonts path and there are no explicit ways to install custom fonts (#2252). I am interested in enhancing matplotlib functionalities (#2270, #300, #2156), but I need some study of matplotlib internals.